### PR TITLE
Fix for Issue#107

### PR DESCRIPTION
**Pull request by nmalasevic**
Fix for issue I reported here:
[https://github.com/gis-ops/routingpy/issues/107](https://github.com/gis-ops/routingpy/issues/107)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -118,6 +118,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - MapboxOSRM directions was parsing geometry to \[lat, lon\]
 - README not valid on PyPI, wasn't rendered
 - add to PyPI and Conda
+- Google router's `duration` and `distance` attributes not being calculated correctly in single route response ([#107](https://github.com/gis-ops/routingpy/issues/107))
+
```
Comment (nilsnolde) [inline CHANGELOG.md:122]: This is the oldest section, v0.2. Can you put at the top under `Unreleased` in a new `Fixed` section?

Comment (nmalasevic) [reply CHANGELOG.md:122]: Sorry, I missed your message, updating it ASAP!

**Comment by nmalasevic** [comment]
I moved it to the correct section. I haven't done that initially since it's still not in master branch, so messed it up :)

File: CHANGELOG.md (hunk 1 above)
Comment (nilsnolde) [reply CHANGELOG.md:126]: But why both now? :sweat_smile: It wasn't released yet, so it only belongs to the Unreleased section

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Valhalla `/expansion` examples in the Jupyter Notebook
 
+## Fixed
```
Comment (nilsnolde) [inline CHANGELOG.md:14]:
```suggestion
### Fixed
```

Comment (nmalasevic) [reply CHANGELOG.md]: Arghhh! No more mistakes!
Let me just squash the commits so there is no noise and it should be ok now.

**Review by nilsnolde — APPROVED** [review]
no worries, we squash by default

File: CHANGELOG.md (hunk 1 above)
Comment (nmalasevic) [reply CHANGELOG.md]: > Unreleased is available in Github's master branch, but not on PyPI.

It wasn't merged to master at the time :smile:

**Pull request merged**